### DISCUSSION TITLE
binary phase diagram

### DISCUSSION THREAD

**Discussion by Liang1Kai** [deepmodeling/dpgen#1290] [General]
I want to build the high temperature part of the binary phase diagram by deep-gen or deep-kit. The potential of binary alloy should be constructed. Whether it is possible to construct full component binary potential functions with deep-gen or deep-kit?
      I would like to ask if there are researchers who have done this work and can provide some relevant suggestions and where should I start.

**Comment by wanghan-iapcm** [comment]
you may want to read https://doi.org/10.1103/PhysRevMaterials.3.023804

**Reply by Liang1Kai** [reply]
Thanks for you reply and suggestion.
